include/Application.h

```diff
@@ -105,10 +105,18 @@ class Application : NoCopy
 #endif // ARCUS
 
     /*!
-     * \brief Print the header and license to the stderr channel.
+     * \brief Print the header to the stderr channel.
+     */
+    void printHeader() const;
+
+    /*!
+     * \brief Print the license to the stderr channel.
      */
     void printLicense() const;
 
+    /*! Print a big warning when the code has been build in debug mode, otherwise just issue a small nice message. */
+    void printDebugWarning() const;
```

Comment:
Is this used anywhere?

Reply:
Hem no, I added the function first but didn't use it and forgot to remove it :expressionless: